internal/gatewayapi/translator.go

```diff
@@ -58,6 +58,32 @@ type Resources struct {
 	Secrets         []*v1.Secret
 }
 
+func (r *Resources) DeepCopy() *Resources {
```

Comment:
instead of hand writing this, have you tried adding the `// +k8s:deepcopy-gen=true` annotation above the struct and running `make generate` ?

Reply:
hey, thanks. I was looking for it, but ended up writing it all and pushing. will do this.